.ci/install_qemu.sh

```diff
@@ -49,7 +49,7 @@ build_static_qemu() {
 uncompress_static_qemu() {
 	local qemu_tar_location="$1"
 	[ -n "$qemu_tar_location" ] || die "provide the location of the QEMU compressed file"
-	sudo tar -xf "${qemu_tar_location}" -C /
+	sudo tar -xf "${qemu_tar_location}" -C ${KATA_QEMU_DESTDIR:-/}
```

Comment:
as we are sourcing `.ci/lib.sh`, `KATA_QEMU_DESTDIR` is pointing to`/usr`, then decompressing qemu to something like: `/usr/usr/bin/qemu...`